spectrum-offchain/src/backlog.rs

```diff
@@ -368,7 +366,7 @@ mod tests {
             order_exec_time: Duration::seconds(order_exec_time_secs),
             retry_suspended_prob: <BoundedU8<0, 100>>::new(retry_suspended_prob).unwrap(),
         };
-        BacklogService::new(store, conf)
+        BacklogService::new::<MockOrder>(store, conf)
```

Comment:
I just fixed up a bunch of compiler complaints about type annotations after I rebased to your last commit.

Reply:
I use VSCode + Rust Analyzer. The error also showed up when running `cargo` from command line.

Reply:
Which version of rust are you using?

Reply:
`cargo build` doesn't give any error in my case

Reply:
1.65, also try `cargo test`

Reply:
Yeah the errors appear in the `tests` module, that's why ti doesn't show up under `build`

Reply:
Oh right